### PR TITLE
166187868 tagging an article

### PR DESCRIPTION
**Pull request by alexxsanya**
#### What does this PR do?
- enable a user to tag an article

#### Description of Task to be completed?

- add functionality to tag an article
- update article test cases

#### How should this be manually tested?
- fetch this repo locally in the terminal with `$ git fetch origin ft-tagging-articles-166187868`
- run `$ python manage.py migrate`
- To tag an article, add the `tagName` attribute to the request body and provide it an array of tags like below.
```JSON
{
	"article":{
		  "title": "string",
		  "description": "string",
		  "body": "string",
		  "image": "http://img.url",
                  "tagName":["tag1","tag2"]
		}
}
```
when creating an article with `POST /api/articles/` & updating an article `PATCH /api/articles/` 
#### What are the relevant pivotal tracker stories?

- Story ID - [166187868](https://www.pivotaltracker.com/story/show/166187868)

### CODE REVIEW

**Review by HabibSentongo — APPROVED** [review]
The feature functions well on my side, I feel this is ready for merge.

File: authors/apps/article_tagging/views.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from authors.apps.article_tagging.serializers import ArticleTaggingSerializer
+
+# Create your views here.
+
+
+class ArticleTaggingViewSet(ModelViewSet):
+
+    queryset = ArticleTagging.objects.all()
+    serializer_class = ArticleTaggingSerializer
+
```
Comment (cdvx) [inline authors/apps/article_tagging/views.py:25]: Please work on these codeclimate issues @alexxsanya

**Review by neelxie — CHANGES_REQUESTED** [review]
Tested the feature and supplying tags like symbols and numbers and it returns an error.

Screenshot 2019-06-21 at 11 50 41

**Comment by alexxsanya** [comment]
> Tested the feature and supplying tags like symbols and numbers and it returns an error.
> 
> Screenshot 2019-06-21 at 11 50 41

@neelxie thanks. this was fixed

**Pull request merged**